component/class/defaults.yml

```diff
@@ -402,6 +406,16 @@ parameters:
         providerConfigRefs: []
         additionalProviderConfigs: []
 
+      sql:
+        enabled: false
```

Comment:
maybe we should not exposed this parameter? `provider-sql` must be available for services that expose user management. What would happen if the sql provider is disabled meanwhile the service postgres allows user management?